Sources/StytchCore/StytchB2BClient/StytchB2BClient+SSO/StytchB2BClient+SSO.swift

```diff
@@ -63,6 +63,14 @@ public extension StytchB2BClient {
             try await router.get(route: .getConnections)
         }
 
+        // sourcery: AsyncVariants, (NOTE: - must use /// doc comment styling)
+        public func getDiscoveryConnections(emailAddress: String) async throws -> GetDiscoveryConnectionsResponse {
```

Comment:
for consistency with other SDKs, can we change this to just `discoverConnections`?